lib/json/schema_builder/schema.rb

```diff
@@ -1,19 +1,39 @@
+require_relative "validation"
+
 module JSON
   module SchemaBuilder
     class Schema
-      attr_accessor :data
+      include Validation
+
+      attr_accessor :data, :entities, :fragments
```

Comment:
is `:fragments` used?

Reply:
Good catch, that was replaced by the fragments method 👍